presentation/backend/src/presentation-backend/PresentationRpcImpl.ts

```diff
@@ -57,14 +57,17 @@ import {
   SelectionScopeRpcRequestOptions,
   SingleElementPropertiesRpcRequestOptions,
 } from "@itwin/presentation-common";
-import { PresentationBackendLoggerCategory } from "./BackendLoggerCategory";
-import { Presentation } from "./Presentation";
-import { PresentationManager } from "./PresentationManager";
-import { TemporaryStorage } from "./TemporaryStorage";
-import { getRulesetIdObject } from "./PresentationManagerDetail";
-
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-const packageJsonVersion = require("../../../package.json").version;
+import { PresentationBackendLoggerCategory } from "./BackendLoggerCategory.js";
+import { Presentation } from "./Presentation.js";
+import { PresentationManager } from "./PresentationManager.js";
+import { TemporaryStorage } from "./TemporaryStorage.js";
+import { getRulesetIdObject } from "./PresentationManagerDetail.js";
+// @ts-expect-error TS complains about `with` in CJS builds; The path is fine at runtime, but not at compile time
+// eslint-disable-next-line @itwin/import-within-package
+import packageJson from "../../../package.json" with { type: "json" };
+
+// const packageJsonVersion = require("../../../package.json").version;
```

Comment:
remove commented out line?

Reply:
yes, thanks for catching this - removed.